### ISSUE TITLE
3.0: Method not found: ProcessForm.downloadToHome()

### ISSUE THREAD

**Issue by mnscholz** [kitodo/kitodo-production#3164]
When clicking the icon "download to home"/"Im Homeverzeichnis verlinken" for a task on the desktop or in the process list, kitodo freezes and the following exception is logged:
`javax.el.MethodNotFoundException: Method not found: org.kitodo.production.forms.ProcessForm@7979591b.downloadToHome()`

Looking into git history, the method was removed in commit 0b529a9, but some xhtml files still call it in the latest commit:
```
grep -r downloadToHome *
...
src/main/webapp/WEB-INF/templates/includes/desktop/processesWidget.xhtml: action="#{DesktopForm.downloadToHome(process.id)}"
** src/main/webapp/WEB-INF/templates/includes/desktop/tasksWidget.xhtml: action="#{ProcessForm.downloadToHome}" **
** src/main/webapp/WEB-INF/templates/includes/processes/processList.xhtml: action="#{ProcessForm.downloadToHome}" **
src/main/webapp/WEB-INF/templates/includes/processes/processList.xhtml: action="#{ProcessForm.downloadToHomeForSelection}"
...
```

Line 2 and 3 use the old function signature.
The first and last line work fine: selecting the process in the process list and choosing the action from the dropdown or clicking on the icon in the process list on the desktop does not result in freeze.

For me, this behaviour shows up both in the 3.0.0 release as well in a recent snapshot.

**Comment by solth** [comment]
I think this is another duplicate of #3313 and therefore has recently been resolved by #3438

**Comment by matthias-ronge** [comment]
It looks like something got mixed up here. Pull Request #3438 actually fixes this problem, but it has nothing to do with issue #3313 (which refers to the **linking** in the user directory while this is about **exporting** to the user home). I close this ticket as solved.

**Comment by Kathrin-Huber** [comment]
Is fixed by #3438
